.github/workflows/test-spras.yml

```
@@ -155,6 +158,18 @@ jobs:
         tags: v2
         cache_froms: reedcompbio/py4cytoscape:v2
         push: false
+  #Adding LN Here
+    - name: Build LocalNeighborhood Docker image
+      uses: docker/build-push-action@v1
+      with:
+        path: docker-wrappers/LocalNeighborhood/.
+        dockerfile: docker-wrappers/LocalNeighborhood/Dockerfile
+        #Ask about the following line
+        repository: reedcompbio/localneighborhood
+        tags: v2
```

```suggestion
        tags: latest
```
We're not versioning this, so latest will be the default tag